packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -201,6 +202,15 @@ export function isIpnMerchPmt(
   return ['merch_pmt', 'mp_cancel'].includes(ipnMessage.txn_type);
 }
 
+type ResponseEventType = {
+  error?: Error;
+  request_end_time: number;
+  version: string;
+  elapsed: number;
+  method: string;
+  request_start_time: number;
```

Comment:
start time, end time, _and_ elapsed seem to be gilding the lily a bit.  Or at least I didn't see where we need the start and end time aside from calculating the elapsed time.

Reply:
Heh, TIL, 'gilding the lily', didn't remember that one!